bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewRepeaterHandler.java

```diff
@@ -79,7 +79,12 @@ public void initialize() {
                     "@text/offline.conf-error.invalid-bridge-handler");
             return;
         }
-        updateStatus(ThingStatus.UNKNOWN);
+        ThingStatus bridgeStatus = bridge.getStatus();
+        if (bridgeStatus == ThingStatus.ONLINE) {
+            updateStatus(ThingStatus.UNKNOWN);
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+        }
         scheduleRefreshJob();
```

Comment:
Do you want to schedule refresh even if bridge is OFFLINE?

Reply:
Good point. This can be optimized, but require some more changes, so outside the scope of this PR. Now that you mention it, I'll reconsider the responsibility between hub handler and repeater handler. In case of more than one repeater this can be optimized if the bridge handler would get data for all repeaters in one call and notify all child repeater handlers.

Reply:
So I merge as it is now in the PR, or not

Reply:
You can merge, this PR just fixes a regression (by reverting a change) caused by a misunderstanding with a PR comment.